src/myuw-help.html

```diff
@@ -1,290 +1,300 @@
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    -webkit-box-align: center;
+    -ms-flex-line-pack: center;
+    -ms-flex-align: center;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+    border-radius: 3px;
+    box-sizing: border-box;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    border: 0;
+    padding: 0 6px;
+    background: transparent;
+    white-space: nowrap;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 14px;
+    text-decoration: none;
+    overflow: hidden;
+    -webkit-transition: box-shadow .4s cubic-bezier(.25, .8, .25, 1), background-color .4s cubic-bezier(.25, .8, .25, 1);
+    transition: box-shadow .4s cubic-bezier(.25, .8, .25, 1), background-color .4s cubic-bezier(.25, .8, .25, 1);
+    border-radius: 50%;
+  }
+
+  #myuw-help__close-button:hover {
+    background-color: rgba(158, 158, 158, 0.2);
+  }
+
+  #help-button {
+    display: none;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+    min-height: 42px;
+    min-width: 42px;
+    height: 42px;
+    width: 42px;
+    user-select: none;
+    outline: none;
+    padding: 0;
+    border: 0;
+    border-radius: 50%;
+    margin: 0 6px;
+    background-color: transparent;
+    -webkit-transition: background-color .3s cubic-bezier(.35, 0, .25, 1);
+    transition: background-color .3s cubic-bezier(.35, 0, .25, 1);
+  }
+
+  #help-icon {
+    color: var(--myuw-primary-color, #fff);
+    fill: var(--myuw-primary-color, #fff);
+    height: 30px;
+    width: 30px;
+  }
+
+  #help-button:hover {
+    background-color: rgba(0, 0, 0, 0.2);
+  }
 
-    @media all and (min-width: 481px) and (max-width: 840px) {
-        #myuw-help__dialog {
-            width: 400px;
-        }
+  #help-button:focus,
+  #help-button:active {
+    outline: 2px solid #5e9ed6;
+    outline: -webkit-focus-ring-color auto 5px;
+  }
+
+  @media all and (min-width: 481px) and (max-width: 840px) {
+    #myuw-help__dialog {
+      width: 400px;
     }
+  }
 
-    @media all and (min-width: 841px) {
-        #myuw-help__dialog {
-            width: 600px;
-        }
+  @media all and (min-width: 841px) {
+    #myuw-help__dialog {
+      width: 600px;
     }
+  }
 </style>
-<button id="help-button" aria-label="open help dialog">
+<button id="help-button" aria-label="Open help dialog">
   <svg id="help-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-    <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 17h-2v-2h2v2zm2.07-7.75l-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25z"/>
-    <path d="M0 0h24v24H0z" fill="none"/>
+    <path
```

Comment:
I would leave this as a single line, since there is only one attribute; in other words, styling it as a block doesn't reveal any extra information to me, esp. as I scan vertically along the left side